helix-view/src/editor.rs

```diff
@@ -190,6 +190,9 @@ pub struct FilePickerConfig {
     /// Enables reading `.git/info/exclude` files.
     /// Whether to hide files listed in .git/info/exclude in file picker and global search results. Defaults to true.
     pub git_exclude: bool,
+    /// Enables use of local and global .gitignore files even if not in a git directory
```

Comment:
This should probably also not be inverse

Reply:
Resolved in both files!